shell/platform/windows/window_binding_handler.h

```diff
@@ -77,6 +77,8 @@ class WindowBindingHandler {
   // Invoked after the window has been resized.
   virtual void OnWindowResized() = 0;
 
+  virtual uint64_t FrameInterval() = 0;
```

Comment:
The type of returned value isn't obvious or documented. Perhaps use `std::chrono::<unit>` and/or add a docstring?